FlowCrypt/Functionality/Services/GlobalRouter.swift

```diff
@@ -95,7 +95,7 @@ extension GlobalRouter: GlobalRouterType {
             }
         } catch {
             logger.logError("Failed to sign out due to \(error.localizedDescription)")
-            handle(error: error, appContext: appContext)
+            hanleFatalError(error)
```

Comment:
@tomholub
Fatal error or splash screen?

Reply:
modal to show the message content, then remain on previous VC?

Reply:
Let's throw it